kubeflow/gcp/iap.libsonnet

```diff
@@ -276,6 +276,7 @@
           service: "backend-updater",
         },
       },
+      serviceName: "backend-updater",
```

Comment:
we don't have such service, will it work?
Same below

Reply:
I have both of these in production. Seems to work fine. As far as I can tell this is the workaround.

It's an open (rotten) issue on the k8s repo.
https://github.com/kubernetes/kubernetes/issues/69608